src/App.js

```diff
@@ -0,0 +1,26 @@
+import { Route, Routes } from 'react-router-dom';
+import React, { Component } from 'react';
+import './App.css';
+import NavBar from './Components/navBar';
+import Books from './Components/books';
+import Categories from './Components/categories';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
+      </>
+    );
```

Comment:
- [ ] it would be better to solve the eslint errors that appear when you navigate to the home page
<img width="1392" alt="Screenshot 2023-01-20 at 16 58 32" src="https://user-images.githubusercontent.com/61790587/213745762-4610b2d3-27bc-4878-bc55-af77f9fad567.png">